src/react-components/avatar-editor.js

```diff
@@ -233,88 +214,123 @@ export default class AvatarEditor extends Component {
     </div>
   );
 
   render() {
-    if (!this.props.signedIn) {
-      return (
-        <div className={classNames(styles.avatarSelectorContainer, "avatar-editor")}>
-          <a onClick={this.props.onSignIn}>
-            <FormattedMessage id="sign-in.in" />
-          </a>
-        </div>
-      );
-    }
-
-    if (!this.state.avatar) {
-      return <div>Loading...</div>;
-    }
-
-    const { debug, preview } = this.props;
-
+    const { debug } = this.props;
     return (
-      <div className={classNames(styles.avatarSelectorContainer, "avatar-editor")}>
-        <div className="split">
-          <div className="form-body">
-            {debug && this.textField("avatar_id", "Avatar ID", true)}
-            {debug && this.textField("parent_avatar_id", "Parent Avatar ID")}
-            {debug && this.textField("name", "Name")}
-            {debug && this.textField("description", "Description")}
-            {debug && this.checkbox("allow_remixing", "Allow Remixing")}
-            {debug && this.checkbox("allow_promotion", "Allow Promotion")}
-            {debug && this.fileField("glb", "Avatar GLB", "model/gltf+binary,.glb")}
-
-            {this.mapField("base_map", "Base Map", "image/*")}
-            {this.mapField("emissive_map", "Emissive Map", "image/*")}
-            {this.mapField("normal_map", "Normal Map", "image/*")}
-
-            {this.mapField("orm_map", "ORM Map", "image/*", false, "Occlussion (r), Roughness (g), Metallic (b)")}
-
-            {/* {this.mapField("ao_map", "AO Map", "images/\*", true)} */}
-            {/* {this.mapField("metallic_map", "Metallic Map", "image/\*", true)} */}
-            {/* {this.mapField("roughness_map", "Roughness Map", "image/\*", true)} */}
+      <div className={classNames(styles.avatarEditor, this.props.className)}>
+        {this.props.onClose && (
+          <a className="close-button" onClick={this.props.onClose}>
+            <i>
+              <FontAwesomeIcon icon={faTimes} />
+            </i>
+          </a>
+        )}
+        {this.props.signedIn ? (
+          <div className="center">
+            <div className="split">
+              <div className="form-body">
+                {debug && this.textField("avatar_id", "Avatar ID", true)}
+                {debug && this.textField("parent_avatar_id", "Parent Avatar ID")}
+                {this.textField("name", "Name", false, true)}
+                {debug && this.textarea("description", "Description")}
+                {debug && this.fileField("glb", "Avatar GLB", "model/gltf+binary,.glb")}
+
+                {this.mapField("base_map", "Base Map", "image/*")}
+                {this.mapField("emissive_map", "Emissive Map", "image/*")}
+                {this.mapField("normal_map", "Normal Map", "image/*")}
+
+                {this.mapField("orm_map", "ORM Map", "image/*", false, "Occlussion (r), Roughness (g), Metallic (b)")}
+
+                {debug && this.checkbox("allow_promotion", "Allow Promotion")}
```

Comment:
Not sure if this would require legal sign off given we have specific [legal copy for Spoke scene promotion](https://github.com/mozilla/Spoke/blob/master/PROMOTION.md). 
Would it make sense to surface promotion alone, or remixing as well? Spoke also has separate legal copy for that. 
Not sure if it should hold this PR up. @gfodor?

Reply:
shouldn't hold it up, i'll surface it in our next meeting but in general we are gtg on avatar customization. promotion/remixing should be off by default for avatars.

Reply:
Unclear if you're saying that this PR *should* show the Allow Promotion and Allow Remixing checkboxes. If it should, shall I just copy and tweak the Scene PROMOTION.md and Scene REMIXING.md files from Spoke?